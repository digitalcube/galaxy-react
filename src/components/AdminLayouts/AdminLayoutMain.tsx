import React , {FC, PropsWithChildren} from 'react'


export type AdminLayoutMainProps = PropsWithChildren<{
    className?: string;
}>

export const AdminLayoutMain: FC<AdminLayoutMainProps> =({children, className}) => {
    return (
        <main
          role="main"
          className={`mb-4 pt-3 px-4 flex-grow-1 shifter-dashboard-main${className ? ` ${className}`: ''}`}
        >
            {children}
        </main>
    )
}