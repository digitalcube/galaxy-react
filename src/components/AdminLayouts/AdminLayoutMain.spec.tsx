import React from 'react';
import { render } from '@testing-library/react';
import { Default as AdminLayoutMain } from './AdminLayoutMain.stories';

describe('AdminLayoutMain', () => {
  it('to match snapshot', () => {
    const { asFragment } = render(
      <AdminLayoutMain>
        <h1>hello</h1>
      </AdminLayoutMain>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
