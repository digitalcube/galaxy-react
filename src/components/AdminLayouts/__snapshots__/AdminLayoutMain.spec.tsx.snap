// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AdminLayoutMain to match snapshot 1`] = `
<DocumentFragment>
  <main
    class="mb-4 px-4 flex-grow-1 shifter-dashboard-main"
    role="main"
  >
    <h1>
      hello
    </h1>
  </main>
</DocumentFragment>
`;
